Yubico.YubiKey/docs/users-manual/application-piv/apdu/verify-uv.md

```diff
@@ -22,9 +22,9 @@ CLA | INS | P1 | P2 | Lc | Data | Le
 00 | 20 | 00 | 96 | *variable* | *variable* | (absent)
 
 The data bytes vary:
-- none  - for checking the biometric state
-- 02 00 - if temporary PIN has been requested
-- 03 00 - if biometric verification has been requested
+- none  - the biometric state has been checked
```

Comment:
I think we should change this a little bit. 
If the instruction is called without any data, it queries the YubiKey for the verification state of biometrics. In the case of unverified state, the YubiKey will respond with `63 CX`, in the case of a verified state `90 00`.
When the state is unverified, the biometrics should not be used, but the PIN verification should be used.

Reply:
I see. But isn't this addressed below in the document on line 55 and onwards?

Reply:
I reviewed the implementation documentation again and the addition in 0d76f0b could be improved, mentioning that:  If all the biometric match retries have been used up, then a 0x6983 error is returned instead. That error is called `AuthenticationMethodBlocked`. 

For the line 25 in this review (where I marked this comment), the line is changed from "for checking the biometric state" to "the biometric state has been checked". I don't think this is accurate as when we call the instruction nothing has been checked yet - we are instructing the YubiKey to response to us what the current verification state of biometrics is.

Reply:
Thanks for catching that, Adam! The tense of the other two lines threw me. 

Changes made here: https://github.com/Yubico/Yubico.NET.SDK/pull/126/commits/ab28b45c9538e101a50214e768e40a639ebef233. Let me know if this covers it.

Reply:
This looks very good now :) Thank you.